clarity/src/vm/costs/mod.rs

```diff
@@ -1194,15 +1194,13 @@ impl CostOverflowingMath<u64> for u64 {
 }
 
 impl ExecutionCost {
-    pub fn zero() -> ExecutionCost {
-        Self {
-            runtime: 0,
-            write_length: 0,
-            read_count: 0,
-            write_count: 0,
-            read_length: 0,
-        }
-    }
+    pub const ZERO: Self = Self {
```

Comment:
You could have done 

```rust
pub const fn zero() { ... }
```

I think using a constant is better practice, but changing it does make the diff larger. No need to change anything, just commenting